### PR TITLE
Task1 PR

### PR DESCRIPTION
**Pull request by fedorenk0**
Did task number one

### CODE REVIEW

File: css/start-use-css/properties-in-css/table.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Properties in CSS</title>
+    <link rel="stylesheet" href="./table.css">
+</head>
+<body>
+    <div>
```
Comment (lutogin) [inline css/start-use-css/properties-in-css/table.html:11]: Why this div is empty

File: css/start-use-css/properties-in-css/task_1.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    width: 300px;
+    height: 60px;
+    color: cyan;
+    background-color: chocolate;
+    border-color: 2px darkgreen;
+    border-radius: 5px;
+    margin: auto;
+    text-align: right;
+    font-style: oblique;
+}
```
Comment (lutogin) [inline css/start-use-css/properties-in-css/task_1.css]: Need new line end of file

Comment (fedorenk0) [reply css/start-use-css/properties-in-css/task_1.css]: Fixed

File: css/start-use-css/properties-in-css/table.html (hunk 1 above)
Comment (fedorenk0) [reply css/start-use-css/properties-in-css/table.html:11]: Its task already in work

**Pull request merged**